src/features/user/UserProfile/components/RedeemModal.tsx

```diff
@@ -119,6 +119,8 @@ export default function RedeemModal({
           setCodeRedeemed(true);
           setIsUploadingData(false);
           setCodeValidated(false);
+          const newUser = {...user, score:{personal: res.schemata.treeCounter.countPlanted}}
```

Comment:
I would explicitly mark this code with a comment to remind ourselves that it needs to be updated once the old API gets replaced with something new.